src/app/content-blocks/block-countdown/block-countdown.component.html

```diff
@@ -24,3 +24,7 @@
     <span>seconds</span>
   </div>
 </div>
+
+<div class="prefix center" *ngIf="hasEnded">
+  The event has ended, see you next time!
```

Comment:
"The event is over, see you next time!" sounds better to me. :P

Reply:
Mayhaps! :)

Reply:
Changed and squashed a fixing commit.